tests_integration/mock_proxy_start.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# If debugging locally, set PROXY_DEBUG_LOGGING to true to show full HTTP request/response details.
+# We can't enable this in GitHub Actions because it will expose GitHub secrets.
+PROXY_DEBUG_LOGGING=${PROXY_DEBUG_LOGGING:-false}
+
+if [[ "${PROXY_DEBUG_LOGGING}" == "true" ]]; then
+  flow_detail=4
+else
+  flow_detail=0
+fi
+
+if [[ -z "$MITMPROXY_CONFDIR" ]]; then
+  echo "Set $MITMPROXY_CONFDIR to a directory containing a mitmproxy-ca.pem CA certificate to intercept HTTPS traffic."
+  exit 1
+fi
+
+# You might need to set PYTHONPATH to the root of cf-scripts
```

Comment:
Can we enforce this with the appropriate working directory?

Reply:
The `PYTHONPATH` is needed because the `mock_server_addon` module needs to be able to import from `tests_integration`, and there is some weird stuff going on in mitmdump. I just tested again, and changing the working directory to the root of cf-scripts does not make it work without `PYTHONPATH`.